web/sidebars.js

```diff
@@ -111,6 +110,12 @@ module.exports = {
       collapsed: false,
       items: ['guides/api'],
     },
+    {
+      type: 'category',
+      label: 'Functions',
+      collapsed: false,
+      items: ['guides/functions/edge-functions', 'guides/database/functions'],
```

Comment:
I've grouped these, because I think conceptually they are the same - the are both Funcitons, just that one runs on your database and one on the edge